client/components/header-cake/style.scss

```diff
@@ -41,6 +41,12 @@
 			margin-right: 4px;
 		}
 	}
+
+	html[dir="rtl"] & {
+		.gridicons-arrow-left {
```

Comment:
Also here: why not simply `html[dir="rtl"] &.gridicons-arrow-left`?

Reply:
I think it's better to keep this in the block, so that it's contained together

Reply:
Alright makes sense. Keep the RTL stuff in a block that allows extending later on.